test/e2e/tests/tokens/token-list.spec.ts

```diff
@@ -155,40 +156,30 @@ describe('Token List', function () {
       async ({ driver }: { driver: Driver }) => {
         await unlockWallet(driver);
         await importToken(driver);
-        await driver.delay(500);
 
         // Verify native token increase
         const testIdNative = `token-increase-decrease-percentage-${zeroAddress()}`;
 
         // Verify native token increase
         const testId = `token-increase-decrease-percentage-${tokenAddress}`;
 
-        const percentageNative = await (
-          await driver.findElement(`[data-testid="${testIdNative}"]`)
-        ).getText();
-        assert.equal(percentageNative, '+0.02%');
-
-        const percentage = await (
-          await driver.findElement(`[data-testid="${testId}"]`)
-        ).getText();
-        assert.equal(percentage, '+0.05%');
-
-        // check increase balance for native token
-        const increaseValue = await (
-          await driver.findElement(
-            `[data-testid="token-increase-decrease-value"]`,
-          )
-        ).getText();
-        assert.equal(increaseValue, '+$50.00 ');
-
-        // check percentage increase balance for native token
-        const percentageIncreaseDecrease = await (
-          await driver.findElement(
-            `[data-testid="token-increase-decrease-percentage"]`,
-          )
-        ).getText();
```

Comment:
There is no timeout for this assertion, we need to give the element sometime to render and fetch.